Registry/openxr.setreg

```diff
@@ -0,0 +1,9 @@
+{
+    "O3DE": {
+        "Atom": {
+            "RHI": {
+                "XRFoveatedLevel" : 2
+            }
+        }
+    }
```

Comment:
we were adding all the XR parameters under `O3DE/Atom/OpenXR`. I'll comment it in the other [PR](https://github.com/o3de/o3de/pull/13811) where this is added.